src/scss/_elements.scss

```diff
@@ -0,0 +1,14 @@
+html {
+  height: 100%;
+  margin:0;
+  padding: 0;
+}
+
+body {
+  font-family: "Montserrat", "Raleway", sans-serif;
+  margin:0;
+  padding: 0;
+  position: relative;
```

Comment:
I'm not sure `position: relative` is necessary here. I tend to only use it on elements whose children need to be absolutely positioned or I need a new stacking context for z-index.

Reply:
I think this is a remnant that should have been deleted. I do have some absolutely position stuff in the project section, but I figured something else out, and I don't think this needs to be here anymore. Thanks for catching that! https://github.com/marissahuysentruyt/marissahuysentruyt/pull/21/commits/c9f02e8ac302925f2cc751214c82d7db449d92ad